data/json/mutations/mutations.json

```diff
@@ -792,6 +792,16 @@
     "category": [ "MEDICAL", "TROGLOBITE" ],
     "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "PAIN", "multiply": -0.33 } ] } ]
   },
+  {
+    "type": "mutation",
+    "id": "CHRONICPAIN",
+    "name": { "str": "Chronic Pain" },
+    "points": -2,
+    "description": "Your entire life you feel a constant pain, all around your body.  Painkillers can relief some of it, but it will never gone away",
```

Comment:
```suggestion
    "description": "For your entire life, you've felt constant pain all around your body.  Painkillers can relieve some of it, but it will never go away.",
```